ts/features/bonusVacanze/store/reducers/bonusVacanze.ts

```diff
@@ -0,0 +1,34 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../store/actions/types";
+import { GlobalState } from "../../../../store/reducers/types";
+import { BonusList } from "../../types/bonusList";
+import { bonusListLoad } from "../actions/bonusVacanze";
+export type BonusState = Readonly<{
+  bonuses: pot.Pot<BonusList, Error>;
+}>;
+
+const INITIAL_STATE: BonusState = {
+  bonuses: pot.none
```

Comment:
Same for the previous comment [here](https://github.com/pagopa/io-app/pull/1843/files#r430331169)